torchbenchmark/score/compute_score_v1.py

```diff
@@ -0,0 +1,193 @@
+"""
+Compute the benchmark score given a frozen score configuration and current benchmark data.
+"""
```

Comment:
Can you explain at the top how many scores are generated and what they mean?

Reply:
There are six scores generated:
1. Total score
2. CPU-train subscore
3. CPU-infer subscore
4. GPU-train subscore
5. GPU-infer subscore
6. JIT speedup score